src/v2/view-builder/views/shared/polling.js

```diff
@@ -2,8 +2,8 @@ import { _ } from 'okta';
 import { MS_PER_SEC } from '../../utils/Constants';
 
 export default {
-  startPolling () {
-    this.pollingInterval = this.options.currentViewState.refresh;
+  startPolling (newInterval) {
+    this.pollingInterval = newInterval || this.options.currentViewState.refresh;
```

Comment:
A further refactoring I can see is always set the interval to `dynamicRefreshInterval`.
1. `in the BaseFormPolling.initialize`, `this.appState.set('dynamicRefreshInterval', .., {silent: true})`
2. then in this method, only read from appState.
3. also need to refractor other views that using polling.

just FYI now. In case you'll be working on this piece of code again, we can try to do it.